src/components/Nav/Nav.scss

```
@@ -0,0 +1,155 @@
+@import '../../styles/mixin.scss';
+
+.nav {
+  position: fixed;
+  width: 100%;
+  height: 80px;
+  background-color: #333;
+  border-bottom: 1px solid #666;
+
+  .depth01 {
+    display: flex;
+    justify-content: space-between;
+    width: 90%;
+    height: 100%;
+    margin: 0 auto;
+    .mainMenu {
+      display: flex;
+      align-items: center;
+      li.logo {
+        margin-right: 1em;
+        a {
+          span {
+            display: block;
+            width: 4em;
+            height: 4em;
+            background: url(../../assets/images/logo_fffef2.png) no-repeat
+              center center / contain;
+            text-indent: -9999px;
+          }
+        }
+      }
```

```suggestion
  .depth01 {
    display: flex;
    justify-content: space-between;
    width: 90%;
    height: 100%;
    margin: 0 auto;
    
    .mainMenu {
      display: flex;
      align-items: center;
      
      li.logo {
        margin-right: 1em;
        
        a {
        
          span {
            display: block;
            width: 4em;
            height: 4em;
            background: url(../../assets/images/logo_fffef2.png) no-repeat
              center center / contain;
            text-indent: -9999px;
          }
        }
      }
```
위코드 프로젝트를 진행하시면서 css 코드 작성할 때 지켜주셔야 하는 컨벤션이 있습니다.
1. selector 의 윗줄은 공백으로 띄워서 가독성을 고려한다.
2. 속성과 속성사이 의미없는 공백은 작성하지 않는다.
요 두가지 컨벤션 잘 지켜서 수정해주세요!